api/actions/signals.py

```diff
@@ -1,20 +1,26 @@
 from django.db.models.signals import post_save, post_delete
 from django.dispatch import receiver
 from .models import ActionFeedback
+import django_rq
 import datetime
 
 
 @receiver(post_save, sender=ActionFeedback, dispatch_uid="UpdateTimeAfterSave")
 @receiver(post_delete, sender=ActionFeedback, dispatch_uid="UpdateTimeAfterDelete")
 def post_save_action_feedback(sender, instance, *args, **kwargs):
+    django_rq.enqueue(update_totals, instance, result_ttl=0)
```

Comment:
We might need to do the same `transaction.on_commit` we do for the notifications. Otherwise I fear we run in the same issue of the new/updated/deleted object not being found when summing the time. 

I didn't notice any issue for the summing and I can't remember how we noticed the issue for the notifications, so not sure how we'd go for testing that. In any case, we can easily re-sum things by going through all volunteers/actions/residents in a script if we notice something.

What do you think?